Ricochet/Assets/_Art/Chris C Particle Effects/Scripts/TriggerCollisionParticle.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace CCParticles
+{
+    public class TriggerCollisionParticle : MonoBehaviour
+    {
+        #region Inspector Variables
+
+        [SerializeField]
+        private ParticleSystem particleSystem;
```

Comment:
overriding a variable in monobehaviour, did you want to change the name of the variable or use the "new" keyword to show that overriding the variable was intended.